packages/modern-poalim-scraper/src/index.ts

```diff
@@ -12,10 +15,10 @@ import {
 } from './scrapers/scrapers-index.js';
 import { newPage } from './utils/browser-util.js';
 
-export async function init() {
+export async function init(headless = true) {
```

Comment:
this should probably be removed before merging

Reply:
Noticed it helped the scraping.
Gets round the bot detection better.
By default it's headless for the other scrapers.
I may need to spend more time masking the browser to get it working in headless mode for Cal. I can check if I'm missing something from Israeli bank scraper project that handles it better.